common/lib/capa/capa/templates/optioninput.html

```diff
@@ -30,6 +30,6 @@
   </div>
   <p class="answer" id="answer_${id}"></p>
   % if msg:
-      <span class="message">${HTML(msg)}</span>
+      <span class="message" aria-describedby="label_${id}" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
Below templates has no label/legend in its own template, but some of them have label/legend in problem.html ${ HTML(problem['html']) }

Do we remove the aria-describeby if we can not use input for id?

1. choice text.html
2. codeinput.html
3. crystallography.html
4. drag_and_drop_input.html
5. file submission.html
6. imageinput.html
7. jsinput.html
8. jstextline.html
9. matlabinput.html
10. schematicinput.html
11. vesper_input.html